securing_apps/topics/oidc/java/spring-security-adapter.adoc

```diff
@@ -164,7 +167,11 @@ While Spring Security's XML namespace simplifies configuration, customizing the
     <security:http auto-config="false" entry-point-ref="keycloakAuthenticationEntryPoint">
         <security:custom-filter ref="keycloakPreAuthActionsFilter" before="LOGOUT_FILTER" />
         <security:custom-filter ref="keycloakAuthenticationProcessingFilter" before="FORM_LOGIN_FILTER" />
+<<<<<<< HEAD
         <security:custom-filter ref="keycloakSecurityContextRequestFilter" after="FORM_LOGIN_FILTER" />
+=======
+	<security:custom-filter ref="keycloakSecurityContextRequestFilter" after="FORM_LOGIN_FILTER" />
+>>>>>>> correction to themes
```

Comment:
I still see the 

> <<<<<<<< HEAD
========
\>>>>>>>> correction to themes

On the file, it is supossed to be there?

```suggestion
        <security:custom-filter ref="keycloakSecurityContextRequestFilter" after="FORM_LOGIN_FILTER" />
```